node/standalone/src/chain_spec.rs

```diff
@@ -39,15 +38,6 @@ pub fn get_from_seed<TPublic: Public>(seed: &str) -> <TPublic::Pair as Pair>::Pu
 		.public()
 }
 
-type AccountPublic = <Signature as Verify>::Signer;
-
-/// Generate an account ID from seed.
-pub fn get_account_id_from_seed<TPublic: Public>(seed: &str) -> AccountId where
-	AccountPublic: From<<TPublic::Pair as Pair>::Public>
-{
-	AccountPublic::from(get_from_seed::<TPublic>(seed)).into_account()
-}
-
```

Comment:
My instinct is that we should keep these helpers (with the new types, of course) and stick with `//Alice` and friends as our pre-funded genesis accounts.

Some benefits of sticking with the default prefunded accounts:
* Sticking closely to Substrate standards
* PolkadotJS Apps knows those accounts and will show them for dev chains (once any type issues are sorted)
* It eliminates the need for us to standardize our own set of accounts, and document them.

Reply:
I agree with the objectives.
We couldn't use Alice and other known account at the beginning because of the mapping and the private key not matching the accountId. 
I believe with the H160, this should be solved on the server side. However I doubt that PolkadotJs would support it. 
My understanding is that Alice, Bob and ... are associated in the Keyring using the sr25519 type, and by doing so won't have the right address.